packages/evmosjs/src/tests/utils/utils.ts

```diff
@@ -0,0 +1,100 @@
+import { Fee, Chain, Sender, TxContext } from '@evmos/transactions'
+import {
+  chainId as networkChainId,
+  cosmosChainId as networkCosmosChainId,
+  denom as networkDenom,
+} from '../network/params'
+
+export class TestingClient {
```

Comment:
Duplicate of @evmos/transactions TestingClient because the other options are worse (expose for export from @evmos/transactions or create a common @evmos/tests package solely for this class)